hack/ci/lib/lib.sh

```diff
@@ -18,26 +18,23 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-_SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
-REPO_ROOT="${_SCRIPT_ROOT}/../../.."
```

Comment:
why was this variable removed? (CI fails on this variable being unbound)

Reply:
🤔 what script is failing? From what I could tell the var was unused 😬

Reply:
Failing script is
```
hack/ci/lib/cluster_create.sh: line 50: REPO_ROOT: unbound variable
```